### ISSUE TITLE
Remove Description Text from Course Boxes in battlefield

### ISSUE THREAD

**Issue by KMankowski** [KMankowski/BattlefieldTSA#20]
Using media queries, remove the descriptive text within boxes on smaller screen sizes. See below.
<img width="196" alt="Screen Shot 2020-08-01 at 8 47 59 AM" src="https://user-images.githubusercontent.com/40276993/89102088-d5fe6280-d3d3-11ea-8e2d-d3a2594dde05.png">

**Comment by Deepakmukka1** [comment]
Ok sure i will work on it

**Comment by Deepakmukka1** [comment]
@KMankowski so you want to get removed the descriptive text completely and only the heading should be?

**Comment by KMankowski** [comment]
Yep, but only on smaller screen sizes! Essentially, it should stay as is on larger screen sizes, but the descriptive text should be removed on smaller screen sizes. The most ideal way to do this would be to set the `<p>` tag containing the descriptive text to `display: none` on smaller screen sizes. You can find out more below:

This project follows the Bootstrap breakpoints:
https://getbootstrap.com/docs/4.1/layout/overview/#responsive-breakpoints

Introduction to media queries: (updating things across screen sizes)
https://www.w3schools.com/css/css3_mediaqueries.asp

How to remove HTML tags/elements on smaller screen sizes:
https://stackoverflow.com/questions/17762354/remove-element-for-certain-screen-sizes
